### PR TITLE
QueryDB

### PR DESCRIPTION
**Pull request by nicola24**
https://trello.com/c/BHZlwg5k

### CODE REVIEW

File: react/src/app.jsx [hunk 1]
```diff
@@ -14,24 +15,29 @@ class App extends React.Component {
   }
 
   componentDidMount() {
-    this.fetch();
+    // this.fetch();
```
Comment (matgbx) [inline react/src/app.jsx:18]: What prompted the change from ajax to axios? Just preference?

**Review by matgbx — APPROVED** [review]
All looks good. Left one comment about axios as I was curious, all good to merge though.

**Review by lee-philip-n — APPROVED** [review]
Looks fine, just a bit worried about the super long src link...

File: react/src/app.jsx (hunk 1 above)
Comment (nicola24) [reply react/src/app.jsx:18]: Axios is just a much better features and in my opinion better overall system than what jQuery has to offer.

**Pull request merged**